jcl/src/jdk.management/share/classes/com/ibm/lang/management/OperatingSystemMXBean.java

```diff
@@ -96,44 +117,6 @@ public interface OperatingSystemMXBean extends com.sun.management.OperatingSyste
 	/*[ENDIF]*/
 	public long getProcessCpuTimeByNS();
 
-	/**
-	 * Returns the recent CPU usage for the whole system. This value is a double in
-	 * the [0.0,1.0] interval. A value of 0.0 means all CPUs were idle in the recent
-	 * period of time observed, while a value of 1.0 means that all CPUs were actively
-	 * running 100% of the time during the recent period of time observed. All values
-	 * between 0.0 and 1.0 are possible. The first call to the method always
-	 * returns {@link com.ibm.lang.management.CpuLoadCalculationConstants}.ERROR_VALUE
-	 * (-1.0), which marks the starting point. If the Java Virtual Machine's recent CPU
-	 * usage is not available, the method returns a negative error code from
-	 * {@link com.ibm.lang.management.CpuLoadCalculationConstants}.
```

Comment:
We should mention CpuLoadCalculationConstants in the class comment as well, outside of the context of z/OS.

Reply:
Done.